frontend/src/pages/search.tsx

```diff
@@ -0,0 +1,21 @@
+import type { GetStaticProps, NextPage } from "next";
+
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+const Search: NextPage = () => {
```

Comment:
I would recommend that the returned Search object is the last thing on the page. Generally React docs recommend that the JSX is the last object on the page for readability.

Reply:
This is how the template repo organized these pages, back when it used nextjs page routing. We'll want to convert to the new app router eventually.